contracts/staking/libs/Rebates.sol

```diff
@@ -86,7 +87,7 @@ library Rebates {
         uint256 rebateReward = 0;
 
         // Calculate the rebate rewards for the indexer
-        if (pool.fees > 0) {
+        if (pool.fees > 0 && pool.effectiveAllocatedStake > 0) {
```

Comment:
Could this be `pool.exists()`? Or, should the definition of `pool.exists` change? (I noticed now it's possible that a pool went through `pool.init` but its effective allocated state is still zero, so not sure how to interpret it or what's the best way to account for it)

Reply:
I feel the more correct thing to do is to redefine exists() as it has more to do with having the alpha initialized, is should have always been like that. wdyt?

Reply:
Yeah that sounds good to me (and it's probably okay to handle in a different PR?)

Reply:
I'll open an issue to tackle in a different PR / upgrade cycle.